Add multiple flag to theme fileupload fields (#64)
wintercms/docs: themes-development.md

});
```

> > **NOTE:** When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship.  If you want to use multiple file uploads for a field, add the `multiple: true` option to the field definition. This will instead add the field to an `attachMany` relationship.

Reviewer: ```suggestion
> **NOTE:** When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship. If you want to use multiple file uploads for a field, add the `multiple: true` option to the field definition. This will instead add the field to an `attachMany` relationship.
```

        default: My Amazing Site!
```

#### Theme attachments

> When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship.  If you want to use multiple file uploads for a field, add the `multiple: true` option to the field definition. This will instead add the field to an `attachMany` relationship.

Author: I would have rather put the note after the configuration with a file.

## Theme customization
Themes can support configuration values ...
### Configuration from a file
You may also define the configuration in a separate file ....
### Important notes
 - jsonables
 - fileuploads
 
I don't find the notes very visible in the documentation, even if they are framed, they should keep the same font size in my opinion.

> When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship.  If you want to use multiple file uploads for a field, add the `multiple: true` option to the field definition. This will instead add the field to an `attachMany` relationship.

Reviewer: I'm fine with the list structure, but it should go right after the fields are first listed, otherwise it could be easily missed.

> When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship.  If you want to use multiple file uploads for a field, add the `multiple: true` option to the field definition. This will instead add the field to an `attachMany` relationship.

Author: I made the change to transpose into a note as requested.

I also made a test of a different structure, take a look and tell me what you think.  
https://gist.github.com/damsfx/d9c877eb40a4638a28387d0c17680ae1

        default: My Amazing Site!
```

#### Theme attachments

When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship.  

> If you want to use multiple files upload for a field (`attachMany` relationship), you have to add the `multiple: true` flag to the field definition.

Reviewer 2: ```suggestion
If you want to use multiple files upload for a field (`attachMany` relationship), you have to add the `multiple: true` option to the field definition.
```

> When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship.  
> If you want to use multiple files upload for a field (`attachMany` relationship), you have to add the `multiple: true` option to the field definition.

Reviewer 3: ```suggestion
When using a `fileupload` field in your theme, Winter will automatically add the field to an `attachOne` relationship.  If you want to use multiple file uploads for a field, add the `multiple: true` option to the field definition. This will instead add the field to an `attachMany` relationship.
```
